apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -123,19 +43,17 @@ export class TeacherFeedback extends Component {
 
     this.onRubricChange = this.onRubricChange.bind(this);
 
+    const {latestFeedback} = this.props;
+
     this.state = {
       comment:
-        this.props.latestFeedback[0] && this.props.latestFeedback[0].comment
-          ? this.props.latestFeedback[0].comment
-          : '',
+        latestFeedback && latestFeedback.comment ? latestFeedback.comment : '',
```

Comment:
I think Charlie added the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) polyfill a while back so we can write the first part as `latestFeedback?.comment`.  If we have the [nullish-coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) polyfill, then we can simplify the whole expression further to `latestFeedback?.comment ?? ''`

Reply:
Woah I didn't know about the nullish-coalescing operator, awesome! Thanks

Reply:
I just got the error `Support for the experimental syntax 'nullishCoalescingOperator' isn't currently enabled`, so I'll use || instead

Reply:
Yeah, I think we might need a polyfill for it but `||` works too!